docs/preview/connected-registry/quickstart-deploy-connected-registry-iot-edge-cli.md

```diff
@@ -164,6 +160,27 @@ To deploy the connected registry module using the Azure CLI, save the following
                         "status": "running",
                         "restartPolicy": "always",
                         "version": "1.0"
+                    },
+                    "IoTEdgeAPIProxy": {
+                        "settings": {
+                            "image": "mycontainerregistry001.azurecr.io/acr/microsoft/azureiotedge-api-proxy:9.9.9-dev",
```

Comment:
mycontainerregistry001.azurecr.io/azureiotedge-api-proxy:9.9.9-dev

Reply:
The command to import the image is az acr import --name mycontainerregistry001 --source acronpremiot.azurecr.io/acr/microsoft/azureiotedge-api-proxy:9.9.9-dev. Should be mycontainerregistry001.azurecr.io/acr/microsoft/azureiotedge-api-proxy:9.9.9-dev?

Reply:
I see you comments on import command and understand what you mean. I will update all of them.

Reply:
Resolved